test/tools/fuzzTesting/fuzzHelper.h

```diff
@@ -121,6 +138,7 @@ class RandomCode
 	static IntDistrib opLengDist;			///< 1..32  byte string
 	static IntDistrib opMemrDist;			///< 1..10MB  byte string
 	static IntDistrib uniIntDist;			///< 0..0x7fffffff
+	static DescreteDistrib opCodeProbability; ///< An Opcode distribution with weights
```

Comment:
Let's try not to introduce a new static variable.

Reply:
would it be better if I declare Random code as single tone?

Reply:
tone?

Reply:
so like all of those fields would be private fields of a class.
and the only static instance would be an instance of that class

Reply:
First, try to remove this line.